cardano/src/redeem.rs

```diff
@@ -110,7 +110,7 @@ impl fmt::Display for PublicKey {
 
 pub const PRIVATEKEY_SIZE: usize = 32;
 
-#[derive(Clone)]
```

Comment:
I'm not sure it makes a lot of sense to define `PartialOrd`/`Ord` for binary hashes; I'd be careful here.